package.json

```diff
@@ -10,7 +10,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/xiaolin/react-image-gallery.git"
+    "url": "https://github.com/lovelysystems/react-image-gallery.git"
```

Comment:
this change makes sure `npm publish` pushes to the correct repository on namespace `lovelysystems`